common/static/js/src/logger.js

```diff
@@ -24,6 +24,9 @@
         return {
             /**
              * Emits an event.
+             *
+             * Note that this method is used by external XBlocks, and the API cannot change without
```

Comment:
These methods have dedicated unit tests in https://github.com/edx/edx-platform/blob/master/common/static/js/spec/logger_spec.js.